snf-image-helper/common.sh.in

```diff
@@ -160,6 +160,9 @@ prepare_helper() {
 initialize_task() {
```

Comment:
Add a comma (`,`) after "enabled" in the commit message or rephrase, e.g. "Disable the unneeded tasks if cloud-init is enabled.".